bionty/models.py

```diff
@@ -237,6 +237,7 @@ def __init__(self, *args, **kwargs):
     def import_source(
         cls,
         source: Source | None = None,
+        version: str | None = None,
```

Comment:
This shouldn't be added, the `import_source` takes a Source record which encodes version already (source.version), once can't pass a version manually.